commands/comment.go

```diff
@@ -58,7 +58,7 @@ func checkCommentLocation(repo repository.Repo, commit, file string, line uint)
 		return err
 	}
 	lines := strings.Split(contents, "\n")
-	if line >= uint(len(lines)) {
+	if line < 1 || line > uint(len(lines)) {
```

Comment:
I actually meant something like:

    if line > uint(len(lines)) {

I.E. You are absolutely right that the check against the number of lines should be `>` rather than `>=`, but there's no need to check that the line is non-negative, as the flag library should not allow that anyway.